pkg/controller/poolcoordinator/cert/poolcoordinatorcert_controller.go

```diff
@@ -175,84 +187,76 @@ var allSelfSignedCerts []CertConfig = []CertConfig{
 	},
 }
 
-// PoolCoordinatorCertManager manages certificates releted with poolcoordinator pod
-type PoolCoordinatorCertManager struct {
-	kubeclientset client.Interface
-	podLister     corelisters.PodLister
-	podSynced     cache.InformerSynced
-	podWorkQueue  workqueue.RateLimitingInterface
+func Format(format string, args ...interface{}) string {
+	s := fmt.Sprintf(format, args...)
+	return fmt.Sprintf("%s: %s", controllerName, s)
 }
 
-func NewPoolCoordinatorCertManager(kc client.Interface, podInformer coreinformers.PodInformer) *PoolCoordinatorCertManager {
-
-	certManager := PoolCoordinatorCertManager{
-		kubeclientset: kc,
-
-		podLister: podInformer.Lister(),
-		podSynced: podInformer.Informer().HasSynced,
-
-		podWorkQueue: workqueue.NewRateLimitingQueue(workqueue.DefaultControllerRateLimiter()),
-	}
+// Add creates a new Poolcoordinatorcert Controller and adds it to the Manager with default RBAC. The Manager will set fields on the Controller
+// and Start it when the Manager is Started.
+func Add(c *appconfig.CompletedConfig, mgr manager.Manager) error {
+	return add(mgr, newReconciler(c, mgr))
+}
 
-	// Watch for poolcoordinator pod changes to manage related certs (including kubeconfig)
-	podInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
-		AddFunc:    func(obj interface{}) {},
-		UpdateFunc: func(oldObj, newObj interface{}) {},
-		DeleteFunc: func(obj interface{}) {},
-	})
+var _ reconcile.Reconciler = &ReconcilePoolcoordinatorcert{}
 
-	return &certManager
+// ReconcilePoolcoordinatorcert reconciles a Poolcoordinatorcert object
+type ReconcilePoolcoordinatorcert struct {
+	Client   client.Interface
+	scheme   *runtime.Scheme
+	recorder record.EventRecorder
```

Comment:
It seems that `scheme` and `recorder` are unused

Reply:
done